modules/@apostrophecms/log/index.js

```diff
@@ -0,0 +1,403 @@
+// Structured logging for Apostrophe.
+//
+// This module is generic, low level implementation. For logging inside of a module,
+// see `logInfo`, `logError`, etc. methods in `@apostrophecms/module`.
```

Comment:
clearer to say "available in every module via the base class, `@apostrophecms/module`"